### DISCUSSION TITLE
IOTA.  I downloaded the Firefly app but the 81-character seed option doesn't show me

### DISCUSSION THREAD

**Discussion by hpereyraesp** [iotaledger/firefly#8221] [Help & Support] [answered]
I downloaded the Firefly app but the 81-character seed option doesn't show me.  Could someone help me?

**Comment by begonaalvarezd** [comment, marked as answer]
There is a dedicated tool to migrate from old 81 chars seed https://blog.iota.org/iota-legacy-migration-tool/

**Reply by hpereyraesp** [reply]
thanks for your help.  It worked.  I beggined the migration process.
